geomesa-convert/geomesa-convert-json/src/main/scala/org/locationtech/geomesa/convert/json/JsonSimpleFeatureConverter.scala

```diff
@@ -243,6 +278,12 @@ trait GeoJsonParsing {
         case "multipolygon" =>
           val polys = el.getAsJsonObject.get(CoordsPath).getAsJsonArray.iterator.map(toPolygon).toArray
           geoFac.createMultiPolygon(polys)
+        case "geometrycollection" =>
+          val gcs = el.getAsJsonObject.get(GeometryPath).getAsJsonArray.iterator.map(toGeometry).toArray
```

Comment:
could this recursively call `parseGeometry` instead of having a separate function?